core-services/release-controller/_releases/release-ocp-4.16-multi.json

```diff
@@ -74,6 +76,27 @@
       "prowJob": {
         "name": "periodic-ci-openshift-multiarch-master-nightly-4.16-ocp-e2e-serial-aws-ovn-heterogeneous"
       }
+    },
+    "multi-gcp-ovn-upgrade": {
+      "optional": true,
+      "prowJob": {
+        "name": "periodic-ci-openshift-multiarch-master-nightly-4.16-ocp-e2e-upgrade-gcp-ovn-heterogeneous"
+      },
+      "upgrade": true,
+      "upgradeFrom": "Previous"
+    },
+    "multi-gcp-ovn-upgrade-from-previous-stable": {
+      "optional": true,
+      "prowJob": {
+        "name": "periodic-ci-openshift-multiarch-master-nightly-4.16-upgrade-from-stable-4.15-ocp-e2e-upgrade-gcp-ovn-heterogeneous"
+      },
+      "upgrade": true,
+      "upgradeFromRelease": {
+        "candidate": {
+          "stream": "nightly-multi",
+          "version": "4.15"
+        }
+      }    
```

Comment:
@jaypoulz These reference jobs that have no corresponding definitions.  Either the jobs must be added or these lines removed.

Reply:
that's correct - reference https://github.com/openshift/release/pull/45417 and copy pasta them for 4.16 I guess

Reply:
Yeah I assumed they already existed and started adding them, but never commit all my local changes. :)

Should be passing now, but I'll give the validation checks a chance to run before I confirm anything.